src/dhcpmon/src/dhcp_device.h

```diff
@@ -49,6 +49,7 @@ typedef struct
     int sock;                       /** Raw socket associated with this device/interface */
     in_addr_t ip;                   /** network address of this device (interface) */
     uint8_t mac[ETHER_ADDR_LEN];    /** hardware address of this device (interface) */
+    in_addr_t loopback_ip;          /** Vlan loopback IP address */
```

Comment:
why named as loopback ip? as far as I know, there is no loopback ip on a vlan.

Reply:
How can I refer to the vlan interface on the router? Would vlan_ip a better match?

Reply:
I think `vlan_ip` makes more sense, unless Guohan has a better idea.

Reply:
I could also use DHCP term giaddr since it is also the agent IP

Reply:
You could use `giaddr` as long as there is a good comment explaining that the Gateway IP address of the device is the IP address of the VLAN interface.